packages/mobile/src/components/image/FastImage.tsx

```diff
@@ -0,0 +1,29 @@
+import type { ImageSourcePropType } from 'react-native'
+import type {
+  FastImageProps as RNFastImageProps,
+  Priority
+} from 'react-native-fast-image'
+import RNFastImage from 'react-native-fast-image'
+
+export type FastImageProps = Partial<Omit<RNFastImageProps, 'source'>> & {
+  source?: ImageSourcePropType
+  priority?: Priority
+  size?: 'small' | 'medium' | 'large'
+}
+
+const sizeMap = {
```

Comment:
Ah ok this is what you meant. I like this as a temp solution! but long term we shouldn't be generating all the extra sources, my pr makes that change

Reply:
Ahh gotcha!